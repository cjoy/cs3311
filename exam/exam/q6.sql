-- COMP3311 12s1 Exam Q6
-- The Q6 view must have attributes called
-- (location,date,team1,goals1,team2,goals2)

drop view if exists Q6;
create view Q6
as
... SQL code for view goes here ...
;
