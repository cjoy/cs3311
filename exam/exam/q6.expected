Santa Cruz|2000-05-24|Argentina|2|France|1
Marseille|2001-03-28|Argentina|2|France|0
Santa Cruz|2001-05-19|Argentina|3|France|0
Marseille|2002-12-06|Argentina|1|France|2
Lyon|2003-02-11|Argentina|1|France|2
Lyon|2003-06-05|Argentina|1|France|2
Santa Cruz|2005-10-04|Argentina|2|France|2
Paris|2006-05-07|Argentina|2|France|2
